src/store/configureStore.js

```diff
@@ -1,14 +1,35 @@
-import { configureStore, createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import {
+  configureStore,
+  createSlice,
+  createAsyncThunk,
+} from '@reduxjs/toolkit';
 
-const fetchGreetings = createAsyncThunk("greetings/fetchGreetings", async () => {});
+const fetchGreetings = createAsyncThunk(
+  'greetings/fetchGreetings',
+  async () => {
+    const response = await fetch(
+      'https://greeting-t0aj.onrender.com/greetings'
+    );
+    const data = await response.json();
+    return data;
+  }
+);
```

Comment:
- [ ] Kindly connect your react app to the api of your Rails application

Reply:
Hey @AshakaE How do you mean connect, when the app us clearly connected here